### ISSUE TITLE
proposal : can define layout for the partail.

### ISSUE THREAD

**Issue by kadevland** [gobuffalo/buffalo#1157]
For partials can define layout for the partials

exemple 
 func (s templateRenderer) partial(name string, dd Data) (template.HTML, error) => 
func (s templateRenderer) partial(name string, dd Data, layout string) (template.HTML, error) ?

question ?:

- the name of real name file of layout with "_" or not ?
- use yield varial our pyield or ... ?

**Comment by markbates** [comment]
Here's my take on what this should look like:

```html
<%= partial("foo.html", {layout: "bar.html"} %>
```

```html
// layout stuff
<%= yield %> // injects partial
// layout stuff
```

**Comment by kadevland** [comment]
thks i will try in next day

**Comment by stanislas-m** [comment]
Fixed with #1201.
